db/data/regions/US/en.yml

```diff
@@ -8,25 +8,40 @@ en:
             default: State
             optional: State (optional)
           errors:
-            blank: Select a state
-            invalid_for_country: Enter a valid %{country} state
-            unknown_for_city_and_zip: Enter a valid state for %{city}, %{zip}
-            unknown_for_city: Select a valid state for %{city}
-            unknown_for_zip: Select a valid state for %{zip}
-            unknown_for_address: State may be incorrect.
-
+            blank_instructional: Select a state
+            blank_informative: Enter a state
+            invalid_for_country_instructional: Enter a valid %{country} state
```

Comment:
Also not a fan of this translation....maybe `Enter a valid state for %{country}%`